common/workunit/workunit.cpp

```diff
@@ -5113,6 +5130,183 @@ bool CWorkUnitFactory::restoreWorkUnit(const char *base, const char *wuid, bool
     return true;
 }
 
+void CWorkUnitFactory::importWorkUnit(const char *zapReportFileName, const char *zapReportFilePath,
```

Comment:
it would be simpler to just pass in the filename, rather than separate out the path/filename and pass in separately.
And it looks like the code checks if paths is empty  and rebuilds path/filename, so it doesn't really want them separate anyway (and could split if did).

Reply:
The filename is separated from the path because the filename will be stored inside the Import Debug Attribute. In the setImportDebugAttribute(), there is a line:
attr.append("ZAPReport=").append(zapReportFileName);

Reply:
I think better to make the use of the interface simple, you can separate the filename from the path inside if needed, e.g. for the above.